<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<testsuite errors="0" failures="2" name="io.cucumber.core.plugin.JUnitFormatter" skipped="0" tests="2" time="3.293">
    <testcase classname="Wallet Company Demo" name="_2" time="3.053">
        <failure message="Step [user take screen shot] is defined with 1 parameters at 'stepDefinitions.Hooks.userTakeScreenShot(java.lang.String)'.&#10;However, the gherkin step has 0 arguments.&#10;Step text: user take screen shot" type="io.cucumber.core.exception.CucumberException"><![CDATA[Given user open "Chrome" browser............................................passed
Then user take screen shot..................................................failed
Then user write the log as "Hello this is log"..............................skipped
Then user get "https://wallethub.com/profile/test-insurance-company-13732055i" url.skipped
And user click the "Reviews_Label" on the "WalletHub_Page"..................skipped
And user move the mouse to the "4thReviewStar_Icon" on the "WalletHub_Page".skipped
And user click the "4thReviewStar_Icon" on the "WalletHub_Page".............skipped
And user click the "Select_DropDown" on the "WalletHub_Page"................skipped
And user click the "HealthInsurance_DropDownValue" on the "WalletHub_Page"..skipped
Given user sendKeys below para in "WriteYourReview_EditBox" on the "WalletHub_Page".skipped

StackTrace:
io.cucumber.core.exception.CucumberException: Step [user take screen shot] is defined with 1 parameters at 'stepDefinitions.Hooks.userTakeScreenShot(java.lang.String)'.
However, the gherkin step has 0 arguments.
Step text: user take screen shot
	at io.cucumber.core.runner.PickleStepDefinitionMatch.arityMismatch(PickleStepDefinitionMatch.java:83)
	at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:42)
	at io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)
	at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:92)
	at io.cucumber.core.runner.TestStep.run(TestStep.java:64)
	at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)
	at io.cucumber.core.runner.TestCase.run(TestCase.java:104)
	at io.cucumber.core.runner.Runner.runPickle(Runner.java:71)
	at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$0(TestNGCucumberRunner.java:116)
	at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:117)
	at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:113)
	at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:31)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
]]></failure>
    </testcase>
    <testcase classname="Wallet Company Demo" name="_3" time="3.025">
        <failure message="Step [user take screen shot] is defined with 1 parameters at 'stepDefinitions.Hooks.userTakeScreenShot(java.lang.String)'.&#10;However, the gherkin step has 0 arguments.&#10;Step text: user take screen shot" type="io.cucumber.core.exception.CucumberException"><![CDATA[Given user open "Chrome" browser............................................passed
Then user take screen shot..................................................failed
Then user write the log as "Hello this is log"..............................skipped
Then user get "https://wallethub.com/profile/test-insurance-company-13732055i" url.skipped
And user click the "Reviews_Label" on the "WalletHub_Page"..................skipped
And user move the mouse to the "4thReviewStar_Icon" on the "WalletHub_Page".skipped
And user click the "4thReviewStar_Icon" on the "WalletHub_Page".............skipped
And user click the "Select_DropDown" on the "WalletHub_Page"................skipped
And user click the "HealthInsurance_DropDownValue" on the "WalletHub_Page"..skipped
Given user sendKeys below para in "WriteYourReview_EditBox" on the "WalletHub_Page".skipped

StackTrace:
io.cucumber.core.exception.CucumberException: Step [user take screen shot] is defined with 1 parameters at 'stepDefinitions.Hooks.userTakeScreenShot(java.lang.String)'.
However, the gherkin step has 0 arguments.
Step text: user take screen shot
	at io.cucumber.core.runner.PickleStepDefinitionMatch.arityMismatch(PickleStepDefinitionMatch.java:83)
	at io.cucumber.core.runner.PickleStepDefinitionMatch.runStep(PickleStepDefinitionMatch.java:42)
	at io.cucumber.core.runner.ExecutionMode$1.execute(ExecutionMode.java:10)
	at io.cucumber.core.runner.TestStep.executeStep(TestStep.java:92)
	at io.cucumber.core.runner.TestStep.run(TestStep.java:64)
	at io.cucumber.core.runner.PickleStepTestStep.run(PickleStepTestStep.java:51)
	at io.cucumber.core.runner.TestCase.run(TestCase.java:104)
	at io.cucumber.core.runner.Runner.runPickle(Runner.java:71)
	at io.cucumber.testng.TestNGCucumberRunner.lambda$runScenario$0(TestNGCucumberRunner.java:116)
	at io.cucumber.core.runtime.CucumberExecutionContext.runTestCase(CucumberExecutionContext.java:117)
	at io.cucumber.testng.TestNGCucumberRunner.runScenario(TestNGCucumberRunner.java:113)
	at io.cucumber.testng.AbstractTestNGCucumberTests.runScenario(AbstractTestNGCucumberTests.java:31)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
	at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
]]></failure>
    </testcase>
</testsuite>
